corehq/apps/domain/models.py

```diff
@@ -121,6 +121,15 @@ def find_cached(self):
     return find_cached
 
 
+def icds_conditional_session_key():
+    if settings.SERVER_ENVIRONMENT == 'icds':
```

Comment:
```suggestion
    if settings.SERVER_ENVIRONMENT in settings.ICDS_ENVS:
```

Reply:
I was intentionally doing this on just ICDS env, since it's likely that settings may be updated frequently on India but not on ICDS

Reply:
`ICDS_ENVS` is currently set to `('icds',)` but it's useful to use it in case we migrate again. All other places use it this way.

Reply:
Cool, will update.

Reply:
Updated